templates/blocks/spreadsheet.twig

```diff
@@ -0,0 +1,25 @@
+{% block spreadsheet %}
+
+	{% if ( fields.rows ) %}
+		<input class="spreadsheet-search form-control" type="text" placeholder="{{ __('Search data') }}"/>
+		<button id="spreadsheet-default-sort">{{ __('Default sorting') }}</button>
```

Comment:
you could add `class="btn btn-secondary"` here to make it look the rest of the buttons

Reply:
I tried the btn-secondary, but it is very big compared to the input and the table. `btn-small` seems to line up better with the search input. But it doesn't have any campaign style yet.

Reply:
![Screenshot from 2019-12-20 12-24-30](https://user-images.githubusercontent.com/7604138/71251967-fc2f3080-2323-11ea-9a58-77aa07d32d08.png)
![Screenshot from 2019-12-20 12-24-01](https://user-images.githubusercontent.com/7604138/71251968-fc2f3080-2323-11ea-9794-a997915c4482.png)

Reply:
Yes, it's indeed very big. So let's keep `btn-small` for now. Since you have a `spreadsheet-default-sort` class there, you could use some css attributes to make it look a bit closer to the rest of the design.

```
.spreadsheet-default-sort {
  border: 1px solid #044362;
  background: #fff;
  color: #044362;
  font-family: Montserrat;
  letter-spacing: .06rem;
  line-height: 38px;
  height: 38px;
  margin-left: 5px;
}

.spreadsheet-default-sort:hover {
  background: #8fd9d0;
  border: 1px solid #8fd9d0;
  color: #044362;
}
```